download_debian_keyring/README.md

```diff
@@ -0,0 +1,51 @@
+# `download_debian_keyring.sh`
+
+## Disclaimer
+
+**Never use the downloaded keyring blindly!**
+
+This script downloads the GPG keys that were used to sign the specified APT repository.
+It does not guarantee that the repository found under the used URL has not been tampered with!
+
+You should always verify the fingerprints of the downloaded keys with the fingerprints publicised by the publisher of the repository.
```

Comment:
```suggestion
You should always verify the fingerprints of the downloaded keys with the fingerprints published by the owner of the repository.
```
Or maybe `creator` instead of owner?